inc/search.php

```diff
@@ -19,15 +19,15 @@
 
 add_action( 'rest_api_init', function() {
 
-	register_rest_route( 'hm-handbook/v1', '/search', array(
+	register_rest_route( 'hm-handbook/v1', '/search', [
```

Comment:
If we're not releasing on the theme repo I don't think it matters. I'm happy to require PHP5.4+